src/utils/hooks.tsx

```diff
@@ -358,7 +377,8 @@ export function useForumData(
     addPost,
     deletePost,
     editPost,
-    update
+    update,
+    fetchVotes,
```

Comment:
Probably shouldn't export fetchVotes here, and instead put the re-fetching logic inside `update()`. That is, unless you explicitly need a function that a user can call to fetch the votes. If that's the case, make another function called `updateVotes()` that calls `fetchVotes`. But `fetchVotes` should just fetch the votes and not have any side effects, if that makes sense. LMK if you want to talk about it